ch.sbb.scion.rcp.feature.test/feature.xml

```diff
@@ -0,0 +1,165 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<feature
+      id="ch.sbb.scion.rcp.feature.test"
+      label="Test"
+      version="1.0.0.qualifier"
+      provider-name="SBB">
+
+   <description url="http://www.example.com/description">
```

Comment:
Are these 3 fields(description, copyright and license) mandatory for a feature.xml? Or just added as placeholders for future completion?

Reply:
You are right. I should either provide useful information or omit it entirely.